omnistrate.enterprise.yaml

```diff
@@ -275,6 +275,14 @@ x-clusterparam: &clusterparam
       defaultValue: "0"
       min: 0
       max: 2000000000
+    - key: maxMemory
+      description: Choose the maxmemory for your database (do not exceed InstanceType memory)
+      name: FalkorDB Maxmemory
+      type: String
+      modifiable: false
+      required: false
+      export: true
+      defaultValue: "0GB"
```

Comment:
Let's set the default to the min we're usually going to deploy which is 8gb - 2gb = 6gb

Reply:
Ok